Sources/Components/HelthjemView/HelthjemView.swift

```diff
@@ -0,0 +1,216 @@
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var detailLabel: Label = {
+        let label = Label(style: .body)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var textContainerView: UIStackView = {
+        let textContainerView = UIStackView(arrangedSubviews: [titleLabel, detailLabel])
+        textContainerView.translatesAutoresizingMaskIntoConstraints = false
+        textContainerView.spacing = .spacingS
+        textContainerView.axis = .vertical
+        return textContainerView
+    }()
+
+    private lazy var primaryButton: Button = {
+        let buttonStyle = Button.Style.flat.overrideStyle(
+            bodyColor: .primaryButtonBackgroundColor,
+            borderWidth: 2,
+            borderColor: .primaryButtonBorderColor,
+            highlightedBodyColor: .primaryButtonBackgroundColor,
+            highlightedBorderColor: .primaryButtonBorderColor
+        )
+        let button = Button(style: buttonStyle)
+        button.backgroundColor = .primaryButtonBackgroundColor
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(handlePrimaryButtonTap), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var secondaryButton: Button = {
+        let buttonStyle = Button.Style.flat.overrideStyle(
+            borderWidth: 0.0,
+            borderColor: nil,
+            textColor: .secondaryButtonColor,
+            highlightedBorderColor: nil,
+            highlightedTextColor: .secondaryButtonColor
+        )
+        let button = Button(style: buttonStyle)
+        button.backgroundColor = .clear
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(handleSecondaryButtonTap), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var buttonContainerView: UIStackView = {
+        let buttonContainerView = UIStackView(arrangedSubviews: [primaryButton, secondaryButton])
+        buttonContainerView.translatesAutoresizingMaskIntoConstraints = false
+        buttonContainerView.spacing = .spacingS
+        return buttonContainerView
+    }()
+
+    private var buttonContainerViewTrailingAnchor = NSLayoutConstraint()
+    private lazy var isIpad = isHorizontalSizeClassRegular
+
+    // MARK: - Init
+
+    public init() {
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = false
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        fatalError()
+    }
+
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        switch previousTraitCollection?.userInterfaceStyle {
```

Comment:
The trait collection can change triggered by other events than change in light/dark mode (e.g. rotation), so I don't think it's safe to use `previousTraitCollection`. You can just use the current `traitCollection` instead 😄 
```
switch traitCollection.userInterfaceStyle {
case .dark: 
      helthjemImage = .init(named: .shipWithHelthjemDarkmode)
default:
     helthjemImage = .init(named: .shipWithHelthjem)
```